Formula/kubescape.rb

```diff
@@ -0,0 +1,36 @@
+class Kubescape < Formula
+  desc "kubescape is the first tool for testing if Kubernetes is deployed securely as defined in Kubernetes Hardening Guidance by to NSA and CISA (https://www.nsa.gov/News-Features/Feature-Stories/Article-View/Article/2716980/nsa-cisa-release-kubernetes-hardening-guidance/) "
+  homepage "https://www.armosec.io/armo-kubescape"
+  url "https://github.com/armosec/kubescape/archive/refs/tags/v1.0.88.tar.gz"
+  sha256 "5d052f1830dcab6e660ca97dade2bcf1b00410197b767060f84f17fcaf43c7cf"
+  license "Apache-2.0"
+
+  bottle do
+      sha256 cellar: :any_skip_relocation, big_sur: "8b6cdcd45abc765d1ffd8dfd5e0bedb9ae4bf93a56f69dab81c0f634ccdb2af5"
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[-s -w
+    -X github.com/armosec/kubescape/cmd.BuildNumber=v#{version.to_s}
+    -X github.com/armosec/kubescape/cautils/getter.ArmoBEURL=api.armo.cloud
+    -X github.com/armosec/kubescape/cautils/getter.ArmoERURL=report.euprod1.cyberarmorsoft.com
+    -X github.com/armosec/kubescape/cautils/getter.ArmoFEURL=portal.armo.cloud
+    ].join(" ")
+
+    system "go", "mod", "tidy"
+    system "go", "build", *std_go_args(ldflags: ldflags)
+
+    output = Utils.safe_popen_read("#{bin}/kubescape", "completion", "bash")
+    (bash_completion/"kubescape").write output
+    output = Utils.safe_popen_read("#{bin}/kubescape", "completion", "zsh")
+    (zsh_completion/"_kubescape").write output
+    output = Utils.safe_popen_read("#{bin}/kubescape", "completion", "fish")
+    (fish_completion/"kubescape").write output
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/kubescape version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I can perform an actual scan with `kubescape scan framework nsa https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/master/release/kubernetes-manifests.yaml
` and check the results.

Reply:
> Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.
> 
> For new formulae:
> 
> > At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.
> 
> For existing formulae:
> 
> > The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.
> 
> Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, adhere to the following guidelines to make sure that @BrewTestBot can merge your commits:
> 
> > * One formula per commit; one commit per formula.
> > * Keep merge commits out of the pull request.
> 
> Also, please run `brew audit --new kubescape` and fix the resulting errors.

`brew audit` seems to be broken on my end. 

```
brew audit --new kubescape
...
Removing rubocop-performance (1.11.5)
Removing sorbet (0.5.9174)
Error: cannot load such file -- rubocop
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/style.rb:91:in `require'
/usr/local/Homebrew/Library/Homebrew/style.rb:91:in `block in run_rubocop'
```

I have to execute `brew update-reset` in order to fix my `brew` again.

Reply:
Able to run `rubocop` separately though.